com.webalys.streamline.yml

```
@@ -0,0 +1,84 @@
+app-id: com.webalys.streamline
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
```

```suggestion
runtime-version: '21.08'
```

19.08 has been EOL since last year (after 2 years)